### ISSUE TITLE
Autoremove bot error messages after a few seconds

### ISSUE THREAD

**Issue by elkowar** [unixporn/trup#57]
when you mistype a command, in many cases the bot prints the usage string. This response should get automatically removed after some time (~10 seconds).
This could also be done for "Success" style responses.
For this, there should probably be a new function `SendAutoremove` (or something like that) created in command.go, which resembles `Reply`, just that it automatically removes the message after a given amount of time.

**Comment by davidv171** [comment]
What's the rationale behind this? Don't people scroll up in #bot to see how other people used the bot, and then they'd be a bit confused about which command is the right or wrong one.

**Comment by elkowar** [comment]
a good idea might be to turn this off in #bot

**Comment by eepykate** [comment]
Yeah, outside of #bot commands like !help, !setfetch, and success/error messages should be automatically deleted to reduce spam.
